src/app/models/BookAnalyzer.java

```diff
@@ -0,0 +1,69 @@
+package models;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class BookAnalyzer {
```

Comment:
name bookAnalyzer sounds like it will provide functionality that will be related to analysis but in this class, we can find methods that display result and save results. It is a violation of single responsibility principle